library/src/main/java/com/novoda/downloadmanager/DownloadBatchRequirementRule.java

```diff
@@ -0,0 +1,17 @@
+package com.novoda.downloadmanager;
+
+/**
+ * Used to define the rules that need to be satisfied to download an batch.
+ * Clients of this library can create their own custom implementations and
+ * pass them to {@link DownloadManagerBuilder#withStorageRequirementRules(StorageRequirementRule...)}.
```

Comment:
Should be linking to -> https://github.com/novoda/download-manager/pull/459/files#diff-bee109b165737f88fe1d3425310bdd9bR171

Reply:
nice catch :)